src/python/python.cpp

```diff
@@ -484,6 +484,7 @@ PYBIND11_MODULE(onnxruntime_genai, m) {
   m.def("is_rocm_available", []() { return USE_ROCM != 0; });
   m.def("is_webgpu_available", []() { return true; });
   m.def("is_qnn_available", []() { return true; });
+  m.def("is_openvino_available", []() { return true; });
```

Comment:
Will it always be available? Or does Onnxruntime have to be built with support for it?

Reply:
Onnxruntime needs to be built with OpenVINO support. I think in the original changeset, it did something similar to rocm case,  `return USE_OPENVINO != 0;`, where `USE_OPENVINO` was introduced by some CMake checks. Perhaps we need a subset of these changes back?

Reply:
alright, please see latest commit. Basically, CMake will check for existence of OpenVINO EP library, and set USE_OPENVINO compile flag accordingly. This is then used within python.cpp to give appropriate return value for `is_openvino_available`

Reply:
Does this mean that it's based on if the machine it was built on has OpenVINO, not the machine it is run on?

Reply:
I think it should always return true. We don't have any compile time dependency and unless we have a runtime way to check if the onnxruntime dll is an openvino one, we should always return true here. It is also how qnn and webgpu is setup.

Reply:
So I think we have 2 options:
1. Implement `is_openvino_available()` and always return True
2. Don't implement it at all

I'm not sure what the intent is of this function, but OpenVINO is available if the following conditions are met :
* OpenVINO onnxruntime EP library is installed into user's env.
* OpenVINO Toolkit environment is installed into user's env.

Perhaps eventually we can implement this function by checking if other packages are installed (onnxruntime-openvino, openvino, etc.) -- but for now, my opinion is to *not* implement it at all (option 2).